packages/@ember/-internals/glimmer/lib/component-managers/curly.ts

```diff
@@ -105,6 +105,19 @@ const EMPTY_POSITIONAL_ARGS: VersionedPathReference[] = [];
 
 debugFreeze(EMPTY_POSITIONAL_ARGS);
 
+function _setupLazyEventsForComponent(dispatcher: any, component: object) {
+  // non-interactive rendering (e.g. SSR) has no event dispatcher
+  if (dispatcher === undefined) { return; }
+
+  let lazyEvents = dispatcher._lazyEvents;
```

Comment:
This also needs to be "semi-public", in terms of `@ember/jquery`, see the other comment...